src/main/resources/openapi/transcriptions.yaml

```diff
@@ -535,6 +591,33 @@ components:
           example: 3
         workflow_comment:
           type: string
+
+    UpdateTranscriptionsRequest:
+      properties:
+        transcriptions:
+            type: array
+            items:
+              $ref: '#/components/schemas/UpdateTranscriptions'
+
+    UpdateTranscriptions:
+      type: object
+      required:
+        - transcriptionId
+      properties:
+        transcriptionId:
+          type: integer
+          example: "1001"
+        hideRequestFromRequestor:
+          type: boolean
+          example: true
+
+    UpdateTranscriptionsResponse:
+      properties:
+        transcriptions:
```

Comment:
Don't think we need this transcriptions property in the Body - it should be fine to just accept an array of objects?

Reply:
removed